mappings/urn.yml

```diff
@@ -0,0 +1,15 @@
+- Frob->Urn:
+    ilkid: frob.ilk
+    urn: frob.urn,
+    ink: frob.ink
+    art: frob.art
+    rat: frob.ink * Ilk.spot / frob.art * Ilk.rate
+    safe: rat > 1
+
+- Bite->Urn:
+    ilkid: bite.ilk
+    urn: bite.urn,
+    ink: 0
```

Comment:
- based on this [conversation](https://chat.makerdao.com/group/vulcanize-ext?msg=vohoL2Takq7PqEhQ4) in the chat, it seems like the `art` and `ink` would be 0 for Bite events. And then since rat is a calculation based on art and ink it would also be 0.